### ISSUE TITLE
Add guide that explains what the render and create function are for

### ISSUE THREAD

**Issue by reactivetype** [debois/elm-parts#6]
Hi @debois 

I learn about this library through elm-mdl and I wish to contribute to the elm-mdl projects. 

However, I could not really understand the Container API functions. I searched through the Slack channel or Elm discuss but I have not found any blog post or guide that explains them. 

It looks to me that the `render` and `create` have a scary signature but I am willing to learn. Perhaps, it would be super useful if you could elaborate what they mean. I went through the simple Counter examples but I am still left wondering about the abstractions in elm-parts. Please elaborate. 

Thank you.

**Comment by debois** [comment]
Hi Rindra, 

Maybe look me up on the elm-slack (@debois), and we can talk?

Cheers,
Søren

**Comment by digital-ember** [comment]
Hi Søren,
not sure if this message was intended for me?
Best,
Robert

2016-07-22 14:27 GMT+02:00 Søren Debois notifications@github.com:

**Comment by debois** [comment]
Not really—I figured you were busy :). But let's talk if you have time! I'm @debois?
